### PR TITLE
Client hints explainer

### PR DESCRIPTION
**Pull request by clelland**
Add an explainer for the relationship between Permissions Policy and Client Hints.

### CODE REVIEW

File: permissions-policy-client-hints.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+documents which it embeds inside of an `<iframe>` element.
+
+By default, low-entropy hints are sent to all embedded documents, while
+high-entropy hints are only sent if the embedded document is same-origin with
+its embedder ([Example 1](#example-1)). The iframe's `allow` attribute is used to
+name specific hints which should be sent to the embedded document
+([Example 2](#example-2)).
+
+> **Question:** Does the embeddded document's `Accept-CH` header affect this
+> processing at all?
```
Comment (yoavweiss) [inline permissions-policy-client-hints.md:71]: It does not

File: permissions-policy-client-hints.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+## Examples
+
+### Example 1: Default behaviour of delegation to frames
+
+Resource: https://example.com/
+
+Response headers:
+```http
+Accept-CH: Sec-CH-DPR
```
Comment (yoavweiss) [inline permissions-policy-client-hints.md:116]: `Accept-CH` doesn't include the `Sec-` and `CH-` prefixes.

**Review by yoavweiss — APPROVED** [review]
LGTM % comments

File: permissions-policy-client-hints.md (hunk 1 above)
Comment (clelland) [reply permissions-policy-client-hints.md]: Removed, thanks.

File: permissions-policy-client-hints.md (hunk 2 above)
Comment (clelland) [reply permissions-policy-client-hints.md]: Prefixes (prefices?) removed, here and elsewhere.

**Pull request merged**